### DISCUSSION TITLE
Directus 9 - using mysql2

### DISCUSSION THREAD

**Discussion by joaocosta-azores** [directus/directus#17658] [Archive]
Hello,
When i try to setup mysql database, i keep getting the error message:
```[ER_NOT_SUPPORTED_AUTH_MODE] ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client```
For what i see, i should be using mysql2 instead of mysql node_module to allow the support for mysql 8 with `caching_sha2_password`.
How can i run directus using `caching_sha2_password` authentication method in mysql8?

Thanks in advance

**Comment by br41nslug** [comment]
It is not currently possible to connect using `caching_sha2_password` authentication as noted in the docs here https://docs.directus.io/self-hosted/docker-guide.html#supported-databases

**Reply by biapar** [reply]
You have to use the standard authentication with MySQL8.

**Reply by sonyarianto** [reply]
any updates on this? this practically not smooth for user that use MySQL 8 want to use Directus.

**Reply by robinsonjohn** [reply]
I am also looking forward to this being supported rather than the less secure `mysql_native_password`.

**Reply by rijkvanzanten** [reply]
Dev is tracked here: https://github.com/directus/directus/issues/19361
